test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -950,6 +950,71 @@ TEST_F(AtenXlaTensorTest, TestLinalgSVD) {
   ExpectCounterChanged("xla::_linalg_svd", cpp_test::GetIgnoredCounters());
 }
 
+TEST_F(AtenXlaTensorTest, TestLinalgVectorNorm) {
```

Comment:
nit: can we add the counters check to verify that the ops are getting properly dispatched to XLA? Example:
```
ExpectCounterNotChanged("aten::.*", cpp_test::GetIgnoredCounters());
ExpectCounterChanged("xla::linalg_vector_norm", cpp_test::GetIgnoredCounters());
```

Reply:
thanks!

Reply:
Let's add it for all the other `linalg_vector_norm` unit tests below as well